features/exploredash/src/main/java/org/dash/wallet/features/exploredash/repository/DataSyncStatus.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Dash Core Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I thought we were using GPL? The year could use an update too :)

Reply:
I copied the wrong license.